osquery/main/tests.cpp

```diff
@@ -9,13 +9,21 @@
  */
 
 #include <cstdlib>
+#include <chrono>
 
 #include <time.h>
 
+#include <osquery/database.h>
+
 #include <gtest/gtest.h>
 
 int main(int argc, char* argv[]) {
-  srand(time(nullptr));
+  std::chrono::milliseconds ms =
+      std::chrono::duration_cast<std::chrono::milliseconds>(
+          std::chrono::system_clock::now().time_since_epoch());
+
+  srand(ms.count());
+  osquery::DBHandle::getInstanceAtPath("/tmp/rocksdb-osquery-dbhandletests");
```

Comment:
This is cool! Can we use /tmp/osquery-tests.db maybe? And right before it `rm /tmp/osquery-tests.db`. No worried if you want to punt on that though.